### PR TITLE
added https option

### PR DESCRIPTION
**Pull request by pieterjandebruyne**
Added possibility to use https when passing a cert and key buffer. (used for https on localhost)

### CODE REVIEW

File: src/RouterServer.ts [hunk 1]
```diff
@@ -106,12 +114,15 @@ export class RouterServer {
...
+        if (this.httpsOptions) {
+            this.server = https.createServer(this.httpsOptions, this.requestListener.bind(this));
+        } else {
+            this.server = http.createServer(this.requestListener.bind(this));
+        }
         this.server.timeout = 10000;
 
         this.server.listen(port, "0.0.0.0", () => {
-            console.log("Server running at http://0.0.0.0:" + port);
+            console.log("Server running at https://0.0.0.0:" + port);
```
Comment (SimonBackx) [inline src/RouterServer.ts:125]: Make a switch between https and http here

**Review by SimonBackx — CHANGES_REQUESTED** [review]
Small thing to fix

File: src/RouterServer.ts (hunk 1 above)
Comment (pieterjandebruyne) [reply src/RouterServer.ts]: fixed

**Review by SimonBackx — APPROVED** [review]

**Comment by SimonBackx** [comment]
Released in 1.4.0

**Pull request merged**